forms.py

```diff
@@ -30,15 +30,25 @@ class ReplyForm(FlaskForm):
     submit = SubmitField("Send message")
 
 class EmployeeForm(FlaskForm):
-    first_name = StringField("First name: ", default="", validators=[InputRequired()])
-    last_name = StringField("Last name: ", default="", validators=[InputRequired()])
-    email = StringField("Email Address", [validators.Length(min=6, max=100)])
+    first_name = StringField("First name ", validators=[InputRequired()])
+    last_name = StringField("Last name", validators=[InputRequired()])
+    email = StringField("Email Address",[validators.Length(min=6, max=100)])
+    address = StringField("Address", default="", validators=[InputRequired()])
```

Comment:
why use default if its just going to be an empty?

Reply:
> why use default if its just going to be an empty?

so true queen slay for noticing HAHAHAHAH